### PR TITLE
Various fixes

### LINKED ISSUE #148: Looking for GJ 3147
Hi Kelle,

I was testing the SIMPLE database looking for this M7V star (also LHS 1378 or LP 245-10) to get its systemic radial velocity but got this "500 something went wrong" error.

Thanks anyway for this huge work

Pascal Fouqué

### PR DESCRIPTION
**Pull request by Will-Cooper**
Changing to @Ajb2307's favicon. Fixing the issue causing server-side error if a user tries to download a non-result, whilst prompting users to raise an issue for missing data. This will additionally close #148. Send the various search options to GET instead of POST, so we can track what users are trying to do. Note this doesn't apply for the coordinate query and sql query, as those require a validation stage. Redirects people using previous bookmarks like fulltextsearch to full_text_search. Reworking of containers on all pages to have more dynamic datatables, improving the general look of the site, most notably the solo_result page.

### CODE REVIEW

**Comment by dr-rodriguez** [comment]
Change to `Please open an issue`

File: simple_app/templates/coordinate_query.html [hunk 1]
```diff
@@ -43,18 +43,25 @@ <h2 class="display-3">Exploring the Database</h2>
...
+    <div class="container table-responsive">
+        {% if query != '' and results is not none %}
         <h2 class="display-3"><a href="{{ url_for('create_file_for_coordinate_download') }}">Results</a>
              for: {{ query|safe }}</h2>
         {% else %}
         <h2 class="display-3">Results for: {{ query|safe }}</h2>
         {% endif %}
         <p class="display-6">{{ results|safe if results is not none }}</p>
+        {% if query != '' %}
+        <p class="display-6">Are we missing an expected result? Please raise an
```
Comment (Will-Cooper) [inline simple_app/templates/coordinate_query.html:55]:
```suggestion
        <p class="display-6">Are we missing an expected result? Please open an
```

File: simple_app/templates/full_text_search.html [hunk 2]
```diff
@@ -73,11 +73,18 @@ <h3 class="display-4">{{ tabname|safe }}</h3>
                 <p class="display-6">{{ df|safe }}</p>
             </div>
         {% endfor %}
+        {% if query != '' %}
+        <p class="display-6">Are we missing an expected result? Please raise an
```
Comment (Will-Cooper) [inline simple_app/templates/full_text_search.html:77]:
```suggestion
        <p class="display-6">Are we missing an expected result? Please open an
```

File: simple_app/templates/raw_query.html [hunk 3]
```diff
@@ -80,18 +80,25 @@ <h3 class="display-4">Examples:</h3>
...
-        {% if query != '' %}
+    <div class="container table-responsive">
+        {% if query != '' and results is not none %}
             <h2 class="display-3"><a href="{{ url_for('create_file_for_sql_download') }}">Results</a>:</h2>
         {% else %}
             <h2 class="display-3">Results:</h2>
         {% endif %}
         <p class="display-6">{{ results|safe if results is not none }}</p>
+        {% if query != '' %}
+        <p class="display-6">Are we missing an expected result? Please raise an
```
Comment (Will-Cooper) [inline simple_app/templates/raw_query.html:91]:
```suggestion
        <p class="display-6">Are we missing an expected result? Please open an
```

File: simple_app/templates/search.html [hunk 4]
```diff
@@ -92,18 +92,25 @@ <h2 class="display-3">Exploring the Database</h2>
...
+    <div class="container table-responsive">
+        {% if query != '' and results is not none %}
         <h2 class="display-3"><a href="{{ url_for('create_file_for_filtered_download') }}">Results</a>
             for: {{ query|safe }}</h2>
         {% else %}
         <h2 class="display-3">Results for: {{ query|safe }}</h2>
         {% endif %}
         <p class="display-6">{{ results|safe if results is not none }}</p>
+        {% if query != '' %}
+        <p class="display-6">Are we missing an expected result? Please raise an
```
Comment (Will-Cooper) [inline simple_app/templates/search.html:104]:
```suggestion
        <p class="display-6">Are we missing an expected result? Please open an
```

File: simple_app/templates/solo_result.html [hunk 5]
```diff
@@ -1,21 +1,23 @@
...
                 </h1>
             </div>
-            <div class="col-md-1">
+            <div class="col-1">
                 <button onclick="share()" type="button" class="btn btn-success"
                         style="float: right" title="Copy link to object">Share</button>
             </div>
         </div>
-        <div class="row">
+        <p class="display-6">Are we missing any expected data? Please raise an
```
Comment (Will-Cooper) [inline simple_app/templates/solo_result.html:18]:
```suggestion
        <p class="display-6">Are we missing any expected data? Please open an
```

**Review by Will-Cooper — COMMENTED** [review]
change 'raise' to 'open'

**Review by dr-rodriguez — APPROVED** [review]

**Pull request merged**